src/Microsoft.AspNet.Mvc.Core/ModelBinding/ComplexModelDtoModelBinder.cs

```diff
@@ -32,7 +32,7 @@ public sealed class ComplexModelDtoModelBinder : IModelBinder
                     await bindingContext.OperationBindingContext.ModelBinder.BindModelAsync(propertyBindingContext);
                 if (modelBindingResult == null)
                 {
-                    // Could not bind. Add a result so MutableObjectModelBinder will check for [DefaultValue].
+                    // Could not bind. Let MutableObjectModelBinder know explicitly.
```

Comment:
Not suggesting to add more to this PR, but the comment suggests this block is unnecessary. IMO correcting the comment is the right thing to do for now though :100: